pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -561,12 +564,15 @@ func manageSchedulerKubeconfig(ctx context.Context, client corev1client.CoreV1In
 
 // getUnsupportedFlagsFromConfig reads and parses flags stored in the "arguments" filed in the unsupported config
 func getUnsupportedFlagsFromConfig(unsupportedConfig []byte) ([]string, error) {
+	if len(unsupportedConfig) == 0 {
```

Comment:
Can you put changes in `getUnsupportedFlagsFromConfig` into https://github.com/openshift/cluster-kube-scheduler-operator/pull/336/commits/4ee3febec88245d9b9769870ccc213117f9e220c given that's the commit where `getUnsupportedFlagsFromConfig` is defined?

Reply:
yeah, I'm going to squash once openshift/library-go#1020 merges